migrations/Version202203151417562260_taoQtiTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use common_ext_Extension as Extension;
+use common_ext_ExtensionsManager as ExtensionsManager;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202203151417562260_taoQtiTest extends AbstractMigration
+{
+    private const CONFIG_FILE = 'testRunner';
+
+    public function getDescription(): string
+    {
+        return 'Set up default config for test-runner review panel';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $extension = $this->getExtension();
+
+        $config = $extension->getConfig(self::CONFIG_FILE);
+        $config['plugins']['review']['reviewLayout'] = 'default';
+        $config['plugins']['review']['displaySectionTitles'] = true;
+        $config['plugins']['review']['displayItemTooltip'] = false;
+        $extension->setConfig(self::CONFIG_FILE, $config);
+
+    }
+
+    public function down(Schema $schema): void
+    {
+        $extension = $this->getExtension();
+
+        $config = $extension->getConfig(self::CONFIG_FILE);
+        unset($config['plugins']['review']);
```

Comment:
I don't know if this full `unset` is ok for the `down` migration... the previous config state must have had `reviewLayout` and `displaySectionTitles` as defined by Siarhei, no?

I suggest unsetting only the new `displayItemTooltip` property.

Reply:
No, they weren't defined, they are defined in this migration.
Unless you think we shouldn't unset it because install script has set them? But then we should never revert config change in migration

Reply:
**nitpick:** I agree with Martin, only the `displayItemTooltip` should be reverted. The 2 other properties were set before. This migration script should only take care of the current state, and move it to the previous. As explained in the other comment, because the previous pull requests that added the base feature did not take care of it, we now have an inconsistency.

Reply:
Ok, I'll do that.
(But again, then we should NEVER revert config change.)